dvc/command/plots.py

```diff
@@ -94,8 +96,12 @@ def run(self):
             )
 
             ui.write(index_path.as_uri())
-
-            if self.args.open:
+            auto_open = to_bool(
+                Config(validate=False)
+                .get("plots", {})
+                .get("auto_open", "false")
+            )
```

Comment:
Why are we re-parsing configs here? This should use the `repo.config` instance:

```py
self.repo.config["plots"].get("auto_open", False)
```

Reply:
Thanks! didn't know. First place I found config usage was https://github.com/iterative/dvc/blob/30d6839a5f471fb02a7a2dcbd6a6616b71277762/dvc/command/daemon.py and just copy from there

Reply:
Is there a reason for doing it differently in `daemon`?

Reply:
It's because it inherits from `CmdBaseNoRepo` (for commands that can be run outside of a DVC repo), so it does not have `self.repo.config`

```
class CmdDaemonBase(CmdBaseNoRepo):
```